src/sentry/rules/actions/notify_event.py

```diff
@@ -28,12 +30,12 @@ def get_plugins(self):
 
         return results
 
-    def after(self, event, state):
+    def after(self, event: Event, state: EventState) -> Any:
```

Comment:
There is a `Generator` type that you could use for the return type here. https://docs.python.org/3/library/typing.html#typing.Generator